akka-actor/src/main/resources/reference.conf

```diff
@@ -59,6 +59,15 @@ akka {
   # setting.
   log-dead-letters-during-shutdown = on
 
+  # Library extensions are regular extensions that are loaded at startup and are
+  # available for third party library authors to enable auto-loading of extensions when
+  # present on the classpath. This is done by appending entries:
+  # 'library-extensions += "Extension"' in the library `reference.conf`.
+  #
+  # Should not be set by end user applications in 'application.conf'.
```

Comment:
also mention something similar to "List FQCN of extensions which shall be loaded at actor system startup."